.github/site-checker/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "site-checker",
```

Comment:
All fields except `scripts` and `devDependencies` can be removed assuming this isn't meant to be published as a global package to the npm registry.

The warnings from npm CLI can be avoided by starting the file with `"private": true,` instead.

Reply:
updated ✅